src/diffusers/models/unet_2d.py

```diff
@@ -105,6 +105,7 @@ def __init__(
         act_fn: str = "silu",
         attention_head_dim: Optional[int] = 8,
         norm_num_groups: int = 32,
+        attn_norm_num_groups: Optional[int] = None,
```

Comment:
Add a docstring entry?

Reply:
I have added a docstring entry for `attn_norm_num_groups`. It looks like none of the `UNetMidBlock2D` arguments have docstrings, so I didn't add one for `attn_groups`.